css/popup.css

```diff
@@ -0,0 +1,249 @@
+body {
+  padding: o;
+  margin: 0;
+}
+
+.popup {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 100;
+  width: 89%;
+  height: 94%;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  padding: 18px 16px 0;
+  border-radius: 16px;
+}
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, I would suggest little touches in your styles!

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/223852565-b1b56c44-3897-4eba-80da-9d8e52c55800.png) | ![image](https://user-images.githubusercontent.com/77069011/223852522-70031294-a9ce-4b7e-bc03-1062c52bee80.png) |

- [ ] 🚧 I think it would be great to add some spacing between the button edges and the end of the popup container, as shown in the FIGMA template. 🤖 

- [ ] 🚧 Also, please make sure you're using the correct `font size` in both, your mobile and desktop versions they're slightly incorrect at the moment, causing some elements, like action buttons (see source) have additional lines. 🤖

Reply:
![image](https://user-images.githubusercontent.com/118625824/223860999-b0fb38b5-70f0-4e8e-a9e7-2ca67466a6cc.png)
i dont know why in your browser its looked like that , in my browser it looked good only i need to fix "see source button"